src/data-seeding/employees/source/default-employees.csv

```diff
@@ -1,12 +1,12 @@
-primaryOfficeId,givenNames,familyName,systemRole,role,mobile,username,email,password
-CRVS_OFFICE_JWMRGwDBXK,Kalusha,Bwalya,FIELD_AGENT,SOCIAL_WORKER,0911111111,k.bwalya,kalushabwalya17@gmail.com,test
-CRVS_OFFICE_JWMRGwDBXK,Felix,Katongo,REGISTRATION_AGENT,REGISTRATION_AGENT,0922222222,f.katongo,kalushabwalya17+@gmail.com,test
-CRVS_OFFICE_JWMRGwDBXK,Kennedy,Mweene,LOCAL_REGISTRAR,LOCAL_REGISTRAR,0933333333,k.mweene,kalushabwalya1.7@gmail.com,test
-CRVS_OFFICE_JWMRGwDBXK,Emmanuel,Mayuka,LOCAL_SYSTEM_ADMIN,LOCAL_SYSTEM_ADMIN,0921681112,e.mayuka,kalushabwalya.17@gmail.com,test
-CRVS_OFFICE_2OKicPQMNI,Jonathan,Campbell,NATIONAL_SYSTEM_ADMIN,NATIONAL_SYSTEM_ADMIN,0921111111,j.campbell,kalushabwaly.a17@gmail.com,test
-CRVS_OFFICE_okQp4uKCz0,Patrick,Gondwe,FIELD_AGENT,LOCAL_LEADER,0912121212,p.gondwe,kalushabwal.ya17@gmail.com,test
-CRVS_OFFICE_okQp4uKCz0,Joshua,Mutale,REGISTRATION_AGENT,REGISTRATION_AGENT,0923232323,j.mutale,kalushabwa.lya17@gmail.com,test
-CRVS_OFFICE_okQp4uKCz0,Derrick,Bulaya,LOCAL_REGISTRAR,LOCAL_REGISTRAR,0934343434,d.bulaya,kalushabw.alya17@gmail.com,test
-CRVS_OFFICE_okQp4uKCz0,Alex,Ngonga,LOCAL_SYSTEM_ADMIN,LOCAL_SYSTEM_ADMIN,0978787878,a.ngonga,kalushab.walya17@gmail.com,test
-CRVS_OFFICE_2OKicPQMNI,Edgar,Kazembe,PERFORMANCE_MANAGEMENT,PERFORMANCE_MANAGER,0977777777,e.kazembe,kalusha.bwalya17@gmail.com,test
-CRVS_OFFICE_2OKicPQMNI,Joseph,Musonda,NATIONAL_REGISTRAR,NATIONAL_REGISTRAR,0915151515,j.musonda,kalush.abwalya17@gmail.com,test
+primaryOfficeId,givenNames,familyName,role,mobile,username,email,password
+CRVS_OFFICE_JWMRGwDBXK,Kalusha,Bwalya,SOCIAL_WORKER,0911111111,k.bwalya,kalushabwalya17@gmail.com,test
+CRVS_OFFICE_JWMRGwDBXK,Felix,Katongo,REGISTRATION_AGENT,0922222222,f.katongo,kalushabwalya17+@gmail.com,test
+CRVS_OFFICE_JWMRGwDBXK,Kennedy,Mweene,LOCAL_REGISTRAR,0933333333,k.mweene,kalushabwalya1.7@gmail.com,test
+CRVS_OFFICE_JWMRGwDBXK,Emmanuel,Mayuka,LOCAL_SYSTEM_ADMIN,0921681112,e.mayuka,kalushabwalya.17@gmail.com,test
+CRVS_OFFICE_2OKicPQMNI,Jonathan,Campbell,NATIONAL_SYSTEM_ADMIN,0921111111,j.campbell,kalushabwaly.a17@gmail.com,test
+CRVS_OFFICE_okQp4uKCz0,Patrick,Gondwe,LOCAL_LEADER,0912121212,p.gondwe,kalushabwal.ya17@gmail.com,test
+CRVS_OFFICE_okQp4uKCz0,Joshua,Mutale,REGISTRATION_AGENT,0923232323,j.mutale,kalushabwa.lya17@gmail.com,test
+CRVS_OFFICE_okQp4uKCz0,Derrick,Bulaya,LOCAL_REGISTRAR,0934343434,d.bulaya,kalushabw.alya17@gmail.com,test
+CRVS_OFFICE_okQp4uKCz0,Alex,Ngonga,LOCAL_SYSTEM_ADMIN,0978787878,a.ngonga,kalushab.walya17@gmail.com,test
+CRVS_OFFICE_2OKicPQMNI,Edgar,Kazembe,PERFORMANCE_MANAGER,0977777777,e.kazembe,kalusha.bwalya17@gmail.com,test
+CRVS_OFFICE_2OKicPQMNI,Joseph,Musonda,LOCAL_REGISTRAR,0915151515,j.musonda,kalush.abwalya17@gmail.com,test
```

Comment:
Do we not have a `NATIONAL_REGISTRAR` anymore?

Reply:
I didn't find any role named `NATIONAL_REGISTRAR`, thus I renamed it like this. 
@rikukissa @jpye-finch